src/processor/FHIRProcessor.ts

```diff
@@ -23,8 +27,11 @@ export class FHIRProcessor {
   process(): Package {
     const resources = new Package();
     let config: ExportableConfiguration;
+    const igForConfig =
+      this.lake.getAllImplementationGuides().find(doc => doc.path.endsWith(this.igPath)) ??
```

Comment:
I don't think `endsWith` works completely here. To test this out, I had a directory with an `ig.ini` contained in a folder, `test/ig.ini`, and then had that `ig.ini` point at an IG file at the same level as the `test` folder. So the path in my `ig.ini` was something like: `..\\ImplementationGuide-hl7.fhir.us.core.json`. But the path in goFSH to the ImplementationGuide file was just `ImplementationGuide-hl7.fhir.us.core.json`, which does not end with `..\\ImplementationGuide-hl7.fhir.us.core.json`, so the ImplemetationGuide is not found.

Reply:
Hm I actually don't think I knew this was allowed, which is probably why I didn't test it. I'm on it!